### PR TITLE
Added new plugin 'drone-azure-pipelines'

### PR DESCRIPTION
**Pull request by localdotcom**

### CODE REVIEW

File: content/localdotcom/drone-azure-pipelines/index.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: Run build pipeline
+  image: dclocal/drone-azure-pipelines:1.0.0
+  settings:
+    action: queue_build
+    definitions: build_definition_id
```
Comment (tboerger) [inline content/localdotcom/drone-azure-pipelines/index.md]: Slice and string in one doesn't make sense

File: content/localdotcom/drone-azure-pipelines/index.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    stages: development
+    organization: organization
+    project: project
+    user: username@domain.com
+    secret:
+      from_secret: az_secret
+```
+
+Example configuration with multiple build pipelines:
+```yaml
```
Comment (tboerger) [inline content/localdotcom/drone-azure-pipelines/index.md]: An an empty line before this block.

File: content/localdotcom/drone-azure-pipelines/index.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
++     - build_definition_id_2
+    organization: organization
+    project: project
+    user: username@domain.com
+    secret:
+      from_secret: az_secret
+```
+
+Example configuration with multiple release pipelines and stages:
+```yaml
```
Comment (tboerger) [inline content/localdotcom/drone-azure-pipelines/index.md]: Same here

File: content/localdotcom/drone-azure-pipelines/index.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
++     - development
++     - test
+    organization: organization
+    project: project
+    user: username@domain.com
+    secret:
+      from_secret: az_secret
+```
+# Parameter Reference
+action
```
Comment (tboerger) [inline content/localdotcom/drone-azure-pipelines/index.md]: Same here

File: content/localdotcom/drone-azure-pipelines/index.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+: Personal Access Token to access REST API
+
+check_build_state (optional)
+: Check the status of associated builds, only available under 'Release' step: true/false
+
+skip (optional)
+: Skip the step execution: true/false 
+
+
+# Hints
```
Comment (tboerger) [inline content/localdotcom/drone-azure-pipelines/index.md]: IMHO these hints should be dropped.

Comment (localdotcom) [reply content/localdotcom/drone-azure-pipelines/index.md]: Hints dropped from code

File: content/localdotcom/drone-azure-pipelines/index.md (hunk 1 above)
Comment (localdotcom) [reply content/localdotcom/drone-azure-pipelines/index.md]: Extra slice removed. My mistake.

File: content/localdotcom/drone-azure-pipelines/index.md (hunk 2 above)
Comment (localdotcom) [reply content/localdotcom/drone-azure-pipelines/index.md]: Empty line removed.

File: content/localdotcom/drone-azure-pipelines/index.md (hunk 3 above)
Comment (localdotcom) [reply content/localdotcom/drone-azure-pipelines/index.md]: Removed

File: content/localdotcom/drone-azure-pipelines/index.md (hunk 4 above)
Comment (localdotcom) [reply content/localdotcom/drone-azure-pipelines/index.md]: Done

File: content/localdotcom/drone-azure-pipelines/index.md (hunk 2 above)
Comment (tboerger) [reply content/localdotcom/drone-azure-pipelines/index.md]: There was a small typo in the message, you should add an empty line around the "headlines".

**Comment by d1wilko** [comment]
Hi,

We have recently updated the drone plugin index and docs are no longer written in markdown - they are now a more consistent yaml format e.g. [docker](https://github.com/drone/drone-plugin-index/blob/master/plugins/docker/content.yaml)

Please have a look at the new readme if you would like to run the new plugin index [readme](https://github.com/drone/drone-plugin-index/blob/master/README.md)

If you would still like to add documentation to the plugins index can you change the format to
- An individual folder per plugin inside the `plugins` folder
- a `content.yaml` file that follows the format in the example above - this is for quick discovery, not full documentation
- an `original.md` file that can contain the markdown you have already written
- a logo image inside the `public/logos` folder

Thanks :)

**Pull request closed without merging**